pkg/sources/github/feedlet/feedlet.go

```diff
@@ -368,13 +368,13 @@ func main() {
 	}
 
 	feedNamespace := os.Getenv(sources.FeedNamespaceKey)
-	if len(feedNamespace) != 0 {
+	if len(feedNamespace) == 0 {
```

Comment:
I have a slight preference for comparing with empty string here, as it seems less likely to produce these sorts of errors.